### DISCUSSION TITLE
Health Checks with http for Sidekiq container

### DISCUSSION THREAD

**Discussion by jonathan-anyword** [sidekiq/sidekiq#6556] [Q&A] [answered]
Hi All,
We are running community version in docker with the command:
`bin/bundle exec sidekiq -t n`

How can I create health check in the container level? our infra needs http response :(
Is it supported on the pro version?
It is verify that Sidekiq is fully active?

Thank you!

**Comment by mperham** [comment]
https://github.com/sidekiq/sidekiq/wiki/Kubernetes#health-checks

**Reply by jonathan-anyword** [reply]
So it only for enterprise? no solution for free or pro?

**Reply by mperham** [reply, marked as answer]
Kubernetes is a complex, enterprisey technology. Hobbyists don't use it.

There are free 3rd party gems for health checks and the page also documents a free solution. If you want an easy solution, be prepared to pay for it.
